elytron/common/src/main/java/org/wildfly/extension/elytron_common/CertificateAuthorityDefinition.java

```diff
@@ -139,13 +148,17 @@ protected TrivialService.ValueSupplier<CertificateAuthority> getValueSupplier(Op
             return () -> {
                 String certificateAuthorityResourceName = context.getCurrentAddress().getLastElement().getValue();
                 return new CertificateAuthority(certificateAuthorityResourceName,
-                        model.get(ElytronDescriptionConstants.URL).asString(),
-                        model.get(ElytronDescriptionConstants.STAGING_URL).asStringOrNull());
+                        model.get(ElytronCommonConstants.URL).asString(),
+                        model.get(ElytronCommonConstants.STAGING_URL).asStringOrNull());
             };
         }
+
+        public String getSubsystemName() {
```

Comment:
Just a minor, this is currently missing Override annotation

Reply:
added